blocklist-client/src/config.rs

```diff
@@ -29,6 +29,8 @@ pub struct RiskAnalysisConfig {
     pub api_url: String,
     /// API key for the Risk service
     pub api_key: String,
+    /// Whether to use sanctions API or RiskAnalysis API
+    pub use_sanctions: bool,
```

Comment:
Nit: I'd rather this be an `assessment_method` enum instead of a boolean where each assessment method has a different enum value.